crates/ruff_db/src/files.rs

```diff
@@ -316,8 +309,8 @@ impl File {
             FilePath::Vendored(_) => {
                 // Readonly, can never be out of date.
             }
-            FilePath::SystemVirtual(system_virtual) => {
-                Self::sync_system_virtual_path(db, &system_virtual, self);
+            FilePath::SystemVirtual(_) => {
+                // TODO(dhruvmanila): Add implementation of sync for virtual files.
             }
```

Comment:
Sorry. This sync function is okay where we already have a file (because it doesn't need looking up the file by path)